File: python/sfo_17_print_from_1_to_the_largest_n_digits_s1.py
'''
File: sfo_17_print_from_1_to_the_largest_n_digits_s1.py
Created Time: 2021-12-09
Author: Krahets (krahets@163.com)
'''

from include import *

# ===== Solution Code =====
class Solution:
    def printNumbers(self, n: int) -> List[int]:
        res = []
        for i in range(1, 10 ** n):
            res.append(i)
        return res

# ======= Test Case =======
n = 1
# ====== Driver Code ======
slt = Solution()
res = slt.printNumbers(n)
print(res)

File: sword_for_offer/codes/java/sfo_17_print_from_1_to_the_largest_n_digits_s1/sfo_17_print_from_1_to_the_largest_n_digits_s1.java
/*
* File: sfo_17_print_from_1_to_the_largest_n_digits_s1.java
* Created Time: 2021-12-09
* Author: Krahets (krahets@163.com)
*/

package sfo_17_print_from_1_to_the_largest_n_digits_s1;

import include.*;
import java.util.*;

// ===== Solution Code =====
class Solution {
    public int[] printNumbers(int n) {
        int end = (int)Math.pow(10, n) - 1;
        int[] res = new int[end];
        for(int i = 0; i < end; i++)
            res[i] = i + 1;
        return res;
    }
}

public class sfo_17_print_from_1_to_the_largest_n_digits_s1 {
    public static void main(String[] args) {
        // ======= Test Case =======
        int n = 1;
        // ====== Driver Code ======
        Solution slt = new Solution();
        int[] res = slt.printNumbers(n);
        System.out.println(Arrays.toString(res));
    }
}
